tests/hw3/test_task4.py

```diff
@@ -0,0 +1,8 @@
+import pytest
+
+from hw3.task4.hw4 import is_armstrong
+
+
+@pytest.mark.parametrize("value, result", [(153, True), (10, False)])
+def test_cache(value: int, result: bool):
```

Comment:
For this function it is ok to use parametrize, but usually such cases should be splitted to two different test functions and be named like
```
def test_is_armstrong_{case_Description}_should_return_true():
   ...
# or simple 
def test_is_armstrong_{case_Description}_true():
   ...
# or simple 
def test_is_armstrong_{case_Description}_should_return_false():
   ...
# or simple 
def test_is_armstrong_{case_Description}_false():
   ...
```
also, forgot to change test function name

Reply:
done